### DISCUSSION TITLE
Can I add custom fonts in Website Builder?

### DISCUSSION THREAD

**Discussion by melpame123** [eclipse/windowbuilder#357] [Q&A]
Hi! I'm fairly new and currently working on this school project where we have to make a website. We are currently using Website Builder and i find the fonts available limited (my focus on this project is on the designing aspect) 

Is it possible to add the custom fonts through the code? My knowledge right now is very limited, so I hope someone can help a newbie like me.

Thank you

<img width="1233" alt="image" src="https://user-images.githubusercontent.com/125524206/219249409-49eb8933-4381-4ebe-a9aa-90ad34b3f1c0.png">

**Comment by wimjongman** [comment]
I asked chatGPT and this was the answer:

`how to change the font of a label in eclipse windowbuilder`

To change the font of a label in Eclipse WindowBuilder, you can follow these steps:

1 Open the WindowBuilder editor and select the label that you want to modify.
2 In the "Properties" view, expand the "Font" property and click on the ellipsis button ("...") next to it. This will open the Font dialog box.
3 In the Font dialog box, you can choose the desired font family, style, and size for the label.
4 Click "OK" to apply the changes and close the Font dialog box.
5 Save your changes by either pressing Ctrl+S or selecting "Save" from the File menu.

Alternatively, you can also change the font of a label programmatically by calling the setFont() method on the label object. For example:

```
java

JLabel myLabel = new JLabel("Hello, world!");
Font font = new Font("Arial", Font.BOLD, 14);
myLabel.setFont(font);
```

This sets the font of the myLabel object to "Arial", bold style, and size 14.
[ChatGPT Feb 13 Version](https://help.openai.com/en/articles/6825453-chatgpt-release-notes)

**Comment by wimjongman** [comment]
This software is not meant to design websites. This will not produce a website. 

However, if you click on lblNewLabel in the content pane, you can change the font in the properties. Please note that there is a different properties view from the one you are showing.

You can press F1 to get to the documentation.
